src/js/actions/ChaosActions.js

```diff
@@ -0,0 +1,103 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+var request = require("../helpers/qajaxWrapper");
+
+var config = require("../config/config");
+
+var AppsStore = require("../stores/AppsStore");
+var ChaosEvents = require("../events/ChaosEvents");
+
+function requestApp(appId, onSuccess, onError) {
+  request({
+    url: config.apiURL + "v2/apps/" + appId
+  })
+  .success(function (data) {
+    onSuccess(data.body.app);
+  })
+  .error(function () {
+    onError();
+  });
+}
+
+function deleteTasks(taskIds = [], onSuccess) {
+  request({
+    method: "POST",
+    data: {
+      "ids": taskIds
+    },
+    headers: {
+      "Content-Type": "application/json"
+    },
+    url: config.apiURL + "v2/tasks/delete"
+  })
+  .success(function () {
+    onSuccess();
+  })
+  .error(function (error) {
+    onSuccess();
+  });
```

Comment:
No, I don't disagree. I am not sure what way to do it right now, but will think about, if I have the time. What way do you prefer? Idented or unintended?
On monday I think you have a coding style discussion scheduled, in persona, so that's why I pointed to the discussion. :)